IS7024Project/Pages/Index.cshtml

```diff
@@ -98,34 +99,16 @@
                 <tr>
                     <td>@crime.IncidentNo</td>
                     <td>@crime.Offense</td>
-                    <td>@crime.Location</td>
+                    <td>@Regex.Replace(@Regex.Replace(@crime.Location.ToString(), "[\\d-]", string.Empty), "([a-z])([A-Z])", "$1 $2")</td>
```

Comment:
First removed the unnecessary and confusing numbers from location then nested in that I added spaces between words for better readability using Regex.

Reply:
Regex is incredibly useful - and equally cryptic.  This is one place where I'd put an inline comment in the code describing the expression.